### ISSUE TITLE
Output csv

### ISSUE THREAD

**Issue by alexn11** [lc5415/COVIDTracker#5]
i got some tweets using backsearch, i ve got issue with reading the data. im using something like:

pd . read_json (path, lines = True) (edited) 

but get an error either

ValueError: Expected object or value

or

ValueError: Unexpected character found when decoding array value (2)

dont know what to do with that.

would it be possible to have csv instead of json? thatd make my life much simpler

**Comment by lc5415** [comment]
@alexn11 for now try:
```
tweets = []
for line in open("path/to/file", "r"):
    tweets.append(json.loads(line))
tweets = pd.DataFrame(tweets)
```

**Comment by lc5415** [comment]
Hello @alexn11, the problem I'm finding with the outputting to csv is that not all tweet have the same number of attributes (e.g. those that have `retweeted_status` as False don't have extra columns about the retweeted tweet). 

json files can handle that better as they are dictionary and can host element of different type and different size. __Is it ok if I do not go on with the csv output issue?__

**Comment by alexn11** [comment]
So I solved my json problem, idk what's the current state of the script SearchBack.py, but when I used that to create my data the json objects were saved all in a single line and json.loads couldnt process this properly. I've sorted this out and created a csv for experiments.
I don't mind using the json format, make sure it can loads with pandas (which apparently it can since make_plot.py seems to work just fine).
